static/resources.json

```diff
@@ -347,6 +347,18 @@
           }
         ]
       },
+      "Python Principles": {
+        "description": "An online Python course for total beginners. Lets you learn through clear explanations and lots of practical exercises. Very hands-on.",
+        "payment": "optional",
```

Comment:
While the free tier provides a glimpse into the service & its offerings, realistically this is a paid service if you want to end with an understanding of Python.